spec/index.bs

```diff
@@ -938,6 +939,12 @@ dictionary GPULimits {
         across all {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+        Higher is [=better=].
+
+    : <dfn>maxUniformBufferBindingSize</dfn>
+    ::
+        The maximum {{GPUBufferBinding}}.{{GPUBufferBinding/size}} for bindings of type {{GPUBindingType/uniform-buffer}}.
```

Comment:
this should be referring to the actual binding size instead of the `size` parameter, to handle the case of size == null. We can follow-up with that

Reply:
Ah yes, that optional is tricky because when `size` is not set, it's no longer clear if it should be `buffer.byteLength - offset` or `min(buffer.byteLength - offset, limits.maxUniformBufferBindingSize)`.

Reply:
Can you leave an `Issue:`?